### PR TITLE
fix the issue of help.sass affecting other pages

### PR DESCRIPTION
**Pull request by DanielLiu14**

### CODE REVIEW

File: constellations/gemini-admin/src/help/help.sass [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  margin: 0
+#view-help
   h1
+    color: $colorTurquoise
     margin-left: 100px
     margin-top: 30px
     font-family: $fontFamilyRoboto
     font-size: $fontSize4
-    color: $colorTurquoise
-
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/help/help.sass:11]: leave the space between properties and selectors

File: constellations/gemini-admin/src/help/help.sass [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
   h2
     font-family: $fontFamilyRoboto
     font-size: $fontSize4
     margin-left: 100px
     padding-bottom: 20px
     border-bottom: 1px solid
     width: 1000px
+
+    
+
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/help/help.sass:24]: hmm i wonder if this is being done by your text editor? Can you remove the extra lines?

File: constellations/gemini-admin/src/help/help.sass [hunk 3]
```diff
@@ -1,24 +1,24 @@
 @import "../_.constants.sass"
 
-body
-  margin: 0
+#view-help
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/help/help.sass:3]: As for formatting, we should try to keep the indents 4 spaces (not 2 which you see in this file). Which code editor are you using? With VSCode there's a way to format your sass files

Comment (DanielLiu14) [reply constellations/gemini-admin/src/help/help.sass:3]: Yeah i'm using VSCode

**Comment by DanielLiu14** [comment]
@ahsu1230 is this good to merge?

**Comment by ahsu1230** [comment]
@tonywu315 I'll defer the review to you here

File: constellations/gemini-admin/src/help/help.sass [hunk 4]
```diff
@@ -1,24 +1,23 @@
...
-    padding-bottom: 20px
-    border-bottom: 1px solid
-    width: 1000px
+    h2
+        font-family: $fontFamilyRoboto
+        font-size: $fontSize4
+        margin-left: 100px
+        padding-bottom: 20px
+        border-bottom: 1px solid
+        width: 1000px
```
Comment (tonywu315) [inline constellations/gemini-admin/src/help/help.sass:23]: There are many predefined lengths in `_constants.sass` already, so you should probably use those. For example, `$spacing128x` is 1024 px.

File: constellations/gemini-admin/src/help/help.sass [hunk 5]
```diff
@@ -1,24 +1,23 @@
...
-    padding-bottom: 20px
-    border-bottom: 1px solid
-    width: 1000px
+    h2
+        font-family: $fontFamilyRoboto
+        font-size: $fontSize4
+        margin-left: $spacing12x
+        padding-bottom: $spacing3x
+        border-bottom: 1px solid
+        width: $spacing128x
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/help/help.sass:23]: Keep CSS ordering: https://github.com/ahsu1230/mathnavigatorSite/blob/master/resources/frontend/css_ordering.md

So I would have width first then margin/padding then border then font related stuff

File: constellations/gemini-admin/src/help/help.sass [hunk 6]
```diff
@@ -1,24 +1,23 @@
...
+        font-size: $fontSize4
     
-  p
-    margin: 20px 0px 40px 100px
-    width: 1000px
-    font-family: $fontFamilyRoboto
-    color: $colorGrayDark
+    p
+        margin: $spacing3x 0px $spacing4x $spacing12x
+        width: $spacing128x
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/help/help.sass:13]: width first in CSS ordering

File: constellations/gemini-admin/src/help/help.sass [hunk 7]
```diff
@@ -1,24 +1,23 @@
...
-    font-family: $fontFamilyRoboto
-    font-size: $fontSize4
-    color: $colorTurquoise
+#view-help
+    h1
+        color: $colorTurquoise
+        margin-left: $spacing12x
+        margin-top: $spacing4x
+        font-family: $fontFamilyRoboto
+        font-size: $fontSize4
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/help/help.sass:9]: margins first: `margin: $spacing4x $spacing12x 0 0`
then font then color

**Comment by ahsu1230** [comment]
Merging this to unblock @srenaxu

**Pull request merged**